controllers/streamController.js

```diff
@@ -35,13 +36,18 @@ const handleNewConnection = async (req, res) => {
 const sendUpdate = (req, res, next) => {
   const updatedFlags = updateFlags(req, [...flags]);
```

Comment:
Does `updateFlags()` work here if it's not defined until later?

Reply:
When does `sendUpdate()` get called / who is calling this? I couldn't find it anywhere